lab_2_keywords_cooccurrence/main.py

```diff
@@ -102,10 +129,13 @@ def calculate_word_scores(word_degrees: Mapping[str, int],
 
     In case of corrupt input arguments, None is returned
     """
-    if not type_check(word_degrees, dict) or not type_check(word_frequencies, dict) \
+    if not is_not_empty_or_bad(word_degrees, dict) or not is_not_empty_or_bad(word_frequencies, dict)\
             or not all(word_frequencies.get(token, False) for token in word_degrees):
         return None
-    return {token: word_degrees[token] / word_frequencies[token] for token in word_degrees}
+    key_for_word_score = list(word_degrees.keys())
+    val_for_word_score = [a / b for a, b in zip(word_degrees.values(), word_frequencies.values())]
+    word_scores = {key_for_word_score[i]: val_for_word_score[i] for i in range(len(key_for_word_score))}
```

Comment:
why convert dictionaries to lists? what if the order of words in these dictionaries is different?